ecms_base/themes/custom/ecms/templates/content/node--event--full.html.twig

```diff
@@ -0,0 +1,145 @@
+ *   - node--promoted: Appears on nodes promoted to the front page.
+ *   - node--sticky: Appears on nodes ordered above other non-sticky nodes in
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ * @todo Remove the id attribute (or make it a class), because if that gets
+ *   rendered twice on a page this is invalid CSS for example: two lists
+ *   in different view modes.
+ */
+#}
+
+{# Start date variables #}
+{% set start_date = node.field_event_date.value|date('F j, Y') %}
+{% set datetime_startdate_format = node.field_event_date.start_value|date('Y-m-d') %}
+
+{# End date variables #}
+{% set end_date = node.field_event_date.end_value|date('F j, Y') %}
+{% set datetime_enddate_format = node.field_event_date.end_value|date('Y-m-d') %}
+
+{# Time variables #}
+{% set start_time = node.field_event_date.value|date('g:ia') %}
+{% set datetime_starttime_format = node.field_event_date.value|date('H:i:s') %}
+{% set end_time = node.field_event_date.end_value|date('g:ia') %}
+{% set datetime_endtime_format = node.field_event_date.end_value|date('H:i:s') %}
+
+{% if start_time != end_time %}
+  {% set time = start_time ~ ' - ' ~ end_time %}
+{% else %}
+  {% set time = start_time %}
+{% endif %}
+
+{# Check for all day #}
+{%  if datetime_starttime_format == '00:00:00' and datetime_endtime_format == '23:59:00' %}
+  {% set allday = TRUE %}
+{% endif %}
+
+{# Generate responsive image style #}
+{% set img = '' %}
+{% if node.field_event_image is not empty %}
+  {% set img_uri = node.field_event_image.entity.field_media_image_1.entity.getFileUri() %}
+  {% set img_alt = node.field_event_image.entity.field_media_image_1.alt %}
```

Comment:
I think we need to check for translation here, i.e. 
```
{% if langcode is null %}
  {% set langcode = 'en' %}
{% endif %}

{% if node.field_event_image.entity.hasTranslation(langcode) %}
      {% set img_uri = node.field_event_image.entity.translation(langcode).field_media_item_image.entity.getFileUri() %}
      {% set img_alt = node.field_event_image.entity.translation(langcode).field_media_item_image.alt %}
```
see `ecms_base/themes/custom/ecms/templates/paragraphs/paragraph--gallery-item.html.twig` and .theme file for langcode in preprocess_ methods (i.e. ecms_preprocess_node__promotions)